File: C#/Practice/LeetCode/Easy/Maximum Sum With Exactly K Elements.cs
// https://leetcode.com/problems/maximum-sum-with-exactly-k-elements/description/

public class Solution {
    public int MaximizeSum(int[] nums, int k) {
    Array.Sort(nums);

    int kSum = 0;

    for (int i = 0; i < k; i++) {
        kSum += i;
    }
    
    return nums[nums.Length - 1] * k + kSum;  
    }
}

File: Python/Practice/LeetCode/Easy/Maximum Sum With Exactly K Elements.py
# https://leetcode.com/problems/maximum-sum-with-exactly-k-elements/description/

class Solution:
    def maximizeSum(self, nums: List[int], k: int) -> int:
        nums.sort()

        kSum = 0

        for i in range(0, k):
            kSum += i

        return nums[len(nums) - 1] * k + kSum